WordPressKit/PostServiceRemoteXMLRPC.m

```diff
@@ -409,6 +412,8 @@ - (NSDictionary *)parametersWithRemotePost:(RemotePost *)post
     if ([post.metadata count] > 0) {
         postParams[@"custom_fields"] = post.metadata;
     }
+    
+    postParams[@"sticky"] = post.isStickyPost ? @"true" : @"false";
```

Comment:
This looks a bit suspect — we're assuming that backend will send us a `1`/`0` value (judging by the `numberForKeyPath` in L323), but we're sending it a string `"true"` and `"false"`?

Reply:
Yes I kept the same behaviour introduced here `PostServiceRemoteREST.m`. Initially I used `1` or `0` but then I switched to `true/false`. Take a look [here](https://github.com/wordpress-mobile/WordPressKit-iOS/pull/31#issuecomment-416931951). I tested `PostServiceRemoteXMLRPC` passing `true/false` and it works.